test/e2e_node/services/kubelet.go

```diff
@@ -93,6 +97,10 @@ const (
 // startKubelet starts the Kubelet in a separate process or returns an error
 // if the Kubelet fails to start.
 func (e *E2EServices) startKubelet() (*server, error) {
+	if kubeletContainerized && hyperkubeImage == "" {
+		return nil, fmt.Errorf("the --hyperkube-image option must be set")
```

Comment:
nit: the --hyperkube-image option must be set if we run kubelet in a docker container

Reply:
The error message pops up only when the ``--containerized`` flag is set. Later, the ``--hyperkube-image`` can point to a container of non-docker container runtime. There is always a space for ambiguity. However, the ``hyperkube`` + ``image`` connection is currently strongly voting for containerized run, the confusion will be minimal. But yeah, updating the message would help to make the statement more clear :).

Reply:
SGTM.